pkg/binding/nested/field_copy.go

```diff
@@ -0,0 +1,23 @@
+package nested
```

Comment:
Let's keep this single function under `binding` package - this opens up possibility to make it private.